Description: Fix setup.py to account for setuptools changes
 Use lists for install_requires and tests_requires.
Author: Andres Rodriguez <andreserl@ubuntu.com>

---
Forwarded: not-needed
Reviewed-By: Andres Rodriguez <andreserl@ubuntu.com>
Last-Update: 2018-02-04

Index: python-libmaas-0.5.0/setup.py
===================================================================
--- python-libmaas-0.5.0.orig/setup.py	2018-02-04 20:55:23.423987036 -0500
+++ python-libmaas-0.5.0/setup.py	2018-02-04 20:55:42.599986962 -0500
@@ -29,7 +29,7 @@
     package_data={
         'maas.client.bones.testing': ['*.json'],
     },
-    install_requires={
+    install_requires=[
         "aiohttp >= 2.0.0",
         "argcomplete >= 1.0",
         "colorclass >= 1.2.0",
@@ -37,16 +37,16 @@
         "pytz >= 2014.10",
         "PyYAML >= 3.11",
         "terminaltables >= 2.1.0",
-    },
+    ],
     test_suite="maas.client",
-    tests_require={
+    tests_require=[
         "django >= 1.6",
         "fixtures >= 1.0.0",
         "setuptools",
         "testscenarios",
         "testtools",
         "Twisted",
-    },
+    ],
     description="A client API library specially for MAAS.",
     entry_points={
         "console_scripts": {
